src/SGM.cpp

```diff
@@ -1,5 +1,28 @@
 #include <SGM.hpp>
 
+namespace IO = EVT::core::IO;
+
 namespace SGM {
 
+SGM::SGM(DEV::StrainGauge gauges[NUM_GAUGES]) : gauges{gauges[0], gauges[1], gauges[2], gauges[3]} {}
+
+CO_OBJ_T* SGM::getObjectDictionary() {
+    return objectDictionary;
 }
+
+uint8_t SGM::getNumElements() {
+    return OBJECT_DICTIONARY_SIZE;
+}
+
+// Update with proper Node ID given by integration!
```

Comment:
This can be updated now with the information here: https://docs.google.com/spreadsheets/d/1O_hIWd3hxgmNijVeCsL_iO469hFsQA3HmNEZrrGsovk/edit?usp=sharing

Reply:
Now that this has been updated, this comment can be deleted